### PR TITLE
added the link for contributing guideline in README

### LINKED ISSUE #3: [DOCS] Contributing Guidelines
I require someone to create CONTRIBUTING GUIDELINES and link the file to the README.md

### PR DESCRIPTION
**Pull request by deeproop**
- Closes #3 

## Type of change
- Updated readme.
- Added contributing guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,17 @@ Currently there are **2** winning solutions, but I plan to add many more questio
 
 You can also run **Choose Your Own Adventure Game** in Gitpod, a free online dev environment for GitHub.
 
-Please make a **PR** if you want to add a **question, fix grammar/typos, etc.** (will add **CONTRIBUTING GUIDELINES soon**).
```
Comment (KendallDoesCoding) [inline README.md:17]: Please link the contributing guidelines.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Please link the contributing guidelines in README

File: README.md (hunk 1 above)
Comment (deeproop) [reply README.md:17]: Please review line 28 of the readme file, where I have added the contributing guideline link.

Comment (KendallDoesCoding) [reply README.md:17]: Sorry, my bad.

File: README.md [hunk 2]
```diff
@@ -14,8 +14,17 @@ Currently there are **2** winning solutions, but I plan to add many more questio
...
 Thanks,
 
 KendallDoesCoding
+
+<!-- CONTRIBUTING -->
+
+## 💁‍♂️Contributing to the project
+Its good to see you contributing to the project.
+
+Check out the [contributing guide](./CONTRIBUTING.md)
```
Comment (KendallDoesCoding) [inline README.md:28]:
```suggestion
Check out our [CONTRIBUTING GUIDELINES](./CONTRIBUTING.md)
```

File: README.md (hunk 1 above)
Comment (deeproop) [reply README.md:17]: No problem :)

**Review by deeproop — COMMENTED** [review]
all good

**Pull request merged**